### DISCUSSION TITLE
What its the correct name?

### DISCUSSION THREAD

**Discussion by Trebol-a** [ultralytics/ultralytics#12848] [Docs]
At launch 8.1.38 with a stream say: 
![image](https://github.com/ultralytics/ultralytics/assets/20790138/3decde94-18c4-487d-9c7f-5a5b3781dbe8)
The documentation page say: 
![image](https://github.com/ultralytics/ultralytics/assets/20790138/15a0b064-c6e6-4a0b-b076-a2bc244c2ed1)

is **stream** or **stream_buffer** ? :\

**Comment by glenn-jocher** [comment]
@Trebol-a hello! Thanks for reaching out. The correct argument to use is `stream_buffer`. It seems there was a discrepancy in the documentation, and we appreciate you bringing this to our attention. We'll update the documentation to reflect the correct usage. If you have any more questions or need further assistance, feel free to ask. Happy coding! 😊

**Comment by glenn-jocher** [comment]
Hello! Thanks for reaching out. The correct argument to use is `stream_buffer`. It seems there's a discrepancy in the documentation, and we appreciate you bringing this to our attention. We'll update the documentation to reflect the correct usage. If you have any more questions or need further assistance, feel free to ask! 🚀
